homeassistant/components/generic/strings.json

```diff
@@ -40,8 +40,11 @@
           "content_type": "Content Type"
         }
       },
-      "confirm": {
-        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      "user_confirm_still": {
+        "description": "Preview\n![Camera Still Image Preview](/api/camera_temp_proxy/{preview_id}/1)",
```

Comment:
Set description to just the image. Set title to "Preview"

Reply:
Done c741557